doc/api/n-api.md

```diff
@@ -105,6 +105,9 @@ typedef enum {
   napi_name_expected,
   napi_function_expected,
   napi_number_expected,
+#ifdef NAPI_EXPERIMENTAL
+  napi_bigint_expected,
```

Comment:
This value should go to the end of the enum, otherwise we break ABI. Once you've moved it to the end of the enum, you also need to append a string explaining the error message to the end of the [array of error messages](https://github.com/nodejs/node/blob/master/src/node_api.cc#L929), otherwise you get a failed assertion.